web/README.md

```diff
@@ -25,6 +25,11 @@ React JS WebApp for [Integreat](https://integreat-app.de) and [Malte](https://ww
 * Run `yarn start` to start a local webpack dev server.
 * Start coding :)
 
+#### Run the App in production mode
+* Create a release build with `yarn build ----env.dev_server`
```

Comment:
```suggestion
#### Run the App in production mode

* Create a release build with `yarn build`
```
Maybe a note `Steps for other build configs differ accordingly`or similar would make sense.